source/guides/api-guidance/index.html.md.erb

```diff
@@ -109,6 +109,15 @@ An OpenAPI definition can then be used by documentation generation tools to disp
 
 Make sure that your API is secure. GDS provides specific guidelines in [API technical and data standards](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#when-to-authenticate-your-api). As well as this general guide, we have provided specific guidance here on securing your DfE API in the [DfE Developer Hub documentation](https://dfe-developerhub.education.gov.uk/documentation/authorisation)
 
+##JSON Security
+
+GDS API standards recommend that [JSON is used](https://www.gov.uk/guidance/gds-api-technical-and-data-standards#use-json), APIs need to be designed so that JSON Objects are submitted rather than individual elements or arrays, JSON data needs to be parsed securely
+* JSON must be well-formed as defined by RFC 4627(http://www.ietf.org/rfc/rfc4627.txt) 
+* Ensure that JSON document structure is valid and matches that expected, for example using a JSONSchema, an OpenAPI specification or deserialising to a Data Transfer Object.
+* Ensure that JSON data elements are valid strong types where possible such as int or [date](https://www.gov.uk/government/publications/open-standards-for-government/date-times-and-time-stamps-standard) 
+* Ensure that JSON data elements that are strings are limited in entry options using a list or REGEX, ensure that your REGEX is well designed so that strings being parsed cannot compromise downstream systems.
```

Comment:
break into two sentences?

Reply:
I think elsewhere we refer to regular expressions as regular expressions or regex - I'm not sure about the use of all upper-case here